### PR TITLE
add new lenient allowlist and consume sum files

### PR DESCRIPTION
**Pull request by ewlu**
Merge only after we have sum files on postcommit

### CODE REVIEW

File: test/benchmarks/dhrystone/dhrystone_main.c [hunk 1]
```diff
@@ -0,0 +1,334 @@
+// See LICENSE for license details.
```
Comment (patrick-rivos) [inline test/benchmarks/dhrystone/dhrystone_main.c:1]: We can remove this file

File: test/benchmarks/dhrystone/dhrystone.h [hunk 2]
```diff
@@ -0,0 +1,469 @@
+// See LICENSE for license details.
```
Comment (patrick-rivos) [inline test/benchmarks/dhrystone/dhrystone.h:1]: We can remove this file

File: test/benchmarks/dhrystone/dhrystone.c [hunk 3]
```diff
@@ -0,0 +1,185 @@
+// See LICENSE for license details.
```
Comment (patrick-rivos) [inline test/benchmarks/dhrystone/dhrystone.c:1]: We can remove this file

File: test/benchmarks/dhrystone/check [hunk 4]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```
Comment (patrick-rivos) [inline test/benchmarks/dhrystone/check:1]: We can remove this file

File: test/allowlist/binutils/glibc.log [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#
```
Comment (patrick-rivos) [inline test/allowlist/binutils/glibc.log:1]: Can remove binutils allowlist

**Review by patrick-rivos — CHANGES_REQUESTED** [review]
Bones look good - I'll take a closer look once the misc files get removed.

**Comment by ewlu** [comment]
#658 with new allowlist
#654 with old allowlist

File: test/allowlist/gcc/glibc.log [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+FAIL: gfortran.dg/maxval_char_1.f90
+FAIL: gfortran.dg/maxval_char_2.f90
+
+#
+# TODO: Root cause & file bugzilla
+#
+FAIL: gcc.target/riscv/zbb-strlen.c
+
+#
+# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=111969
```
Comment (patrick-rivos) [inline test/allowlist/gcc/glibc.log:20]: These fails only occur on rv32gcv. We can use a more targeted allowlist file here (we can also just merge a minimal allowlist and have a separate PR to add the flaky fails to that)

**Review by patrick-rivos — APPROVED** [review]
LGTM! I can send another PR to update the allowlist with the fails I've noticed.

**Pull request merged**